apps/client/src/app/layout.tsx

```diff
@@ -1,20 +1,22 @@
-import "./globals.css";
+import './globals.css';
```

Comment:
we probably think introducing format check in husky, this kind of format style should not be seen in PR @NorrisWu0